content/webapp/components/ExhibitionCaptions/ExhibitionCaptions.tsx

```diff
@@ -259,9 +249,11 @@ const Stop: FunctionComponent<{
                   {title}
                 </TombstoneTitle>
               )}
-              <div className={font('intr', 4)}>
-                <PrismicHtmlBlock html={tombstone} />
-              </div>
+              {tombstone && (
+                <div className={font('intr', 4)}>
+                  <PrismicHtmlBlock html={tombstone} />
+                </div>
+              )}
```

Comment:
I don't think empty tombstones were adding any spacing we needed; I've compared the IPS guide locally and in prod and they look the same.